nss-day-cohort-46/reactive-nutshell-talisman-of-havocing
# Nc building weather provider

# Description
built the weather provider that takes a city name and state to get data for that location
## Type of change


# Testing Instructions
replace the data in your json for events with the one that is in the example file. Open the events tab. Check the console that shows the weather data. It will look like a big array of jumbled data but thats the weather.

# YOU WILL NEED A SETTINGS.JS FILE NOW. This will go in the auth directory. add this file to your .gitignore.
Copy and paste this inside your settings.js file:
export const settings = {
	weatherKey: "pasteYourOpenWeatherAPIKeyFromHolidayRoadHere"
};

# Checklist:
